sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -77,7 +77,7 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
    * <li>`overwrite`: overwrite the existing data.</li>
    * <li>`append`: append the data.</li>
    * <li>`ignore`: ignore the operation (i.e. no-op).</li>
-   * <li>`error` or `errorifexists`: default option, throw an exception at runtime.</li>
+   * <li>`error`, `errorifexists`, or `default`: default option, throw an exception at runtime.</li>
```

Comment:
actually, I think it is a bit weird to give a "default" option explicitly. I'd tend to remove "default" option, if possible.

Reply:
I also want to hide this `default` from the document. In any way, we need to fix the code.